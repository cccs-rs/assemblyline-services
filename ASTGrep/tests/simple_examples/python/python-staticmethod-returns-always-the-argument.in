class lI1lI1l1lI1l1lll1II:
    @staticmethod
    def llIIIIl1(llIllllI):
        return llIllllI

lI1lI1l1lI1l1lll1II.llIIIIl1(123)
