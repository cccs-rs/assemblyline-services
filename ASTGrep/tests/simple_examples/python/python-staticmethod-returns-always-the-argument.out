class lI1lI1l1lI1l1lll1II:
    @staticmethod
    def llIIIIl1(llIllllI):
        return llIllllI

123
